3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,104 @@
+ //hello... Hello... This is my first experience in Javascript :)
+
+console.log("******    Age-ify (A future age calculator)    ******");
+let myYearOfBirth = 1991;
+let yearFuture = 2027;
+let age = yearFuture - myYearOfBirth;
+
+console.log("I\'ll be " + age + " years old in " + yearFuture + ".");
+console.log("\n");
+
+
+
+console.log("******    Goodboy-Oldboy (A dog age calculator)    ******");
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2045;
+// We have a conditional statement, so a "BOOLEAN variable" can represent (shouldShowResultInDogYears).
+let shouldShowResultInDogYears = true;
+
+ // According to the information in the txt:(1 human year = 7 dog year)
+let dogAge = (dogYearFuture - dogYearOfBirth)*7;
+let humanAge = dogYearFuture - dogYearOfBirth;
+
+if (shouldShowResultInDogYears) {
+    console.log("His dog will be " + dogAge + " dog years old in " + dogYearFuture + ".");
+}
+
+else {
+ console.log("His dog will be " + humanAge + " human years old in " + dogYearFuture + ".");
+}
+console.log("/* His dog will be 28 human years old in 2045. */");
+console.log("\n");
+
+
+console.log("******    Housey pricey (A house price estimator)    ******");
+//Peter:
+var petersPreferredHouseWide = a;
+var petersPreferredHouseDeep = b;
+var petersPreferredHouseHigh = c;
+
+let petersGardenSizeInM2 = 100; //m2
+let petersPreferredHouseCosts = 2500000; //$
+
+var a = 8; //m
+var b = 10; //m
+var c = 10; //m
+
+var volumeInMeters = a * b * c;
+const petersHousePrice = volumeInMeters * 2.5 * 1000 + petersGardenSizeInM2 * 300;
+const petersPriceDifference = petersPreferredHouseCosts - petersHousePrice;
```

Comment:
I think, that you should check whether the price is too high or the price is good using a condition (same as with `dogYears` assignment)

something like 
```
if(housePrice > petersHousePrice) {
 // too expensive } else {
 // good deal }
```

Reply:
many thanks for that much attention. 
Done!